source/include/FreeRTOS_IPv6_Sockets.h

```diff
@@ -102,8 +102,7 @@
  *         is connected to a remote IP-address. It will be called from a loop
  *         iterating through all sockets. */
     FreeRTOS_Socket_t * pxTCPSocketLookup_IPv6( FreeRTOS_Socket_t * pxSocket,
-                                                const IPv6_Address_t * pxAddress_IPv6,
-                                                uint32_t ulRemoteIP );
+                                                const IPv46_Address_t * pxAddress );
```

Comment:
We already have correct IP types passed in both functions, If we are not making the function as one function then what is the point of updating the arguments. The end goal of updating the arguments and making it common should be to get rid of one of the function.

Reply:
We need a method to differentiate if the address is belong to IPv4 or IPv6. An dthe new added structure `IPv46_Address_t` has a bit for it.